lib/ReactViews/Analytics/BooleanParameterEditor.jsx

```diff
@@ -11,31 +12,47 @@ const BooleanParameterEditor = React.createClass({
     },
 
     onClick() {
-        const value = this.props.previewed.parameterValues[this.props.parameter.id];
-        this.props.previewed.setParameterValue(this.props.parameter.id, !value);
+        this.props.parameter.value = !this.props.parameter.value;
+    },
+
+    renderCheckbox() {
+        const value = this.props.parameter.value;
+        const name = this.props.parameter.name;
+        const description = this.props.parameter.description;
+
+        return (
+            <div>
+                <button type='button'
+                        className={Styles.btnRadio}
+                        title={description}
+                        onClick={this.onClick}>
+                    {value && <Icon glyph={Icon.GLYPHS.checkboxOn}/>}
+                    {!value && <Icon glyph={Icon.GLYPHS.checkboxOff}/>}
+                    {name}
+                </button>
+            </div>
+        );
```

Comment:
Sure, that makes sense. We should add a boolean parameter for WPS at some stage, although I guess no one has asked for it yet. I'm happy with this then. Thanks!